src/interpolations.jl

```diff
@@ -57,6 +57,10 @@ n_components(::VectorInterpolation{vdim}) where {vdim} = vdim
 # Number of components that are allowed to prescribe in e.g. Dirichlet BC
 n_dbc_components(ip::Interpolation) = n_components(ip)
 
+shape_value_type(::ScalarInterpolation, T::Type) = T
```

Comment:
Add a docstring to this one in devdocs for interpolations, and filter the one in [docs/src/devdocs/FEValues.md](https://github.com/Ferrite-FEM/Ferrite.jl/blob/212115a3e041ac19be7c78fe50b2e8a251aa03d3/docs/src/devdocs/FEValues.md?plain=1#L25) to show for `AbstractValues` and `FunctionValues` inputs?